tests/integration/heartbeat/heartbeat.spec.ts

```diff
@@ -90,5 +94,43 @@ describe('heartbeat', function () {
 
       expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
     });
+
+    it('post heartbeat should return 400 if given heartbeat id is not uuid, ', async () => {
+      const badId = '1';
+
+      const response = await requestSender.pulse(badId);
+      expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
+    });
+  });
+
+  describe('GetHeartbeat', function () {
+    describe('Happy path', function () {
+      it('if heartbeat exist for id, should return 200 with heartbeat row', async () => {
+        const id = '720d1a04-5a9e-4a3e-beba-6fe980fca516';
+
+        await requestSender.pulse(id);
+        const response = await requestSender.getHeartbeat(id);
+        expect(response.body.id).toBe(id);
+        expect(new Date(response.body.lastHeartbeat)).toBeInstanceOf(Date);
+
+        requestSender.removeHeartbeats([id]);
+      });
+    });
+
+    describe('Sad path', function () {
```

Comment:
"Sad path" mostly refers to 500+ error codes.
```suggestion
    describe('Bad path', function () {
```